frontend/sw360-portlet/src/main/webapp/html/homepage/myprojects/view.jsp

```diff
@@ -25,25 +25,24 @@
 <div id="myProjectsDiv" class="homepageListingTable">
     <table id="myProjectsTable" cellpadding="0" cellspacing="0" border="0" class="display">
          <colgroup>
-               <col style="width: 60%;"/>
                <col style="width: 40%;"/>
+               <col style="width: 30%;"/>
+               <col style="width: 30%;"/>
          </colgroup>
     </table>
 </div>
 
 <script>
-
     Liferay.on('allPortletsReady', function() {
         var result = [];
-
         <core_rt:forEach items="${projects}" var="project">
         result.push({
             "DT_RowId": "${project.id}",
             "0": "<sw360:DisplayProjectLink project="${project}"/>",
-            "1": '<sw360:out value="${project.description}"/>'
+            "1": '<sw360:out value="${project.description}"/>',
+            "2": '<sw360:out value="${project.releaseClearingStateSummary.approved} / ${project.releaseClearingStateSummary.newRelease + project.releaseClearingStateSummary.underClearing + project.releaseClearingStateSummary.underClearingByProjectTeam + project.releaseClearingStateSummary.reportAvailable + project.releaseClearingStateSummary.approved}" default="--"/>'
```

Comment:
Maybe create a sw360:tag to calculate this.
We already have a similar tag to show this information, its called `DisplayReleaseClearingStateSummary`.
This tag is not used and maybe you can modify it.

Also beware of null values in `project.releaseClearingStateSummary` in the error case becuase you only return the project list without release clearing state summary information.

Reply:
Now this is calculated in `DisplayAcceptedReleases` which is modified from `DisplayReleaseClearingStateSummary`